src/Grpc.Net.Client/Internal/GrpcEventSource.cs

```diff
@@ -0,0 +1,162 @@
+﻿#region Copyright notice and license
+
+// Copyright 2019 The gRPC Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#endregion
+
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+#if HAVE_DIAGNOSTICS_COUNTERS
+using System.Threading;
+#endif
+using Grpc.Core;
+
+namespace Grpc.Net.Client.Internal
+{
+    internal class GrpcEventSource : EventSource
+    {
+        public static readonly GrpcEventSource Log = new GrpcEventSource();
+
+#if HAVE_DIAGNOSTICS_COUNTERS
+        private PollingCounter? _totalCallsCounter;
```

Comment:
Is PollingCounter available outside of .NET Core 3.0? Current this package only targets netstandard2.1

Reply:
Nope. Counters are not in netstandard (at least not yet) cc @terrajobst

Reply:
What about an external package? I tried to find a package that contained PollingCounter and couldn't find one.

Reply:
They're in netstandard2.1 now 🎉